README.md

```diff
@@ -1,9 +1,55 @@
 # JupyterHub Administration to Empower Research Communities
 
-This lesson teaches the fundamentals of JupyterHub administration and user support.
+This lesson is aimed at Hub Champions and teaches the fundamentals of hub administration and user support for 2i2c managed JupyterHubs in the cloud. Delivered over two synchronous workshops, two hours in length a week apart. The time in between is dedicated to self-guided study with asynchronous support (via email 2i2c support / online Slack? To be discussed).
 
-## Note 
-This lesson is under active development and not ready for teaching yet.
+> [!WARNING]
+> This lesson is under active development and not ready for teaching yet.  
+
+> [!INFO]
+> TODO: Link to the Spanish repo.
+
+## Lesson objectives
+
+After attending this training, Hub Champions will be able to:
+
+- explain how interactive cloud computing can empower your user community
+- control user authentication and access to your hub
+- monitor usage and cloud billing costs
+- apply software environments through containerization
+- navigate the filesystem and transfer data to and from the hub
+- provide user support and troubleshoot common issues
+
+## Episode objectives
+
+1. :star: Recognising the benefits your hub can provide for your user community
+   - identify use cases of your user community
+   - distinguish which hub features can support these use cases
+   - understand your role and capacity as a Hub Champion to administer and support your hub
+1. Administering your hub
+   - add users to the hub with GitHub
+   - configure server options (CPU, GPU, RAM)
+   - stop and restart a server
+   - monitor usage and estimate cloud billing costs
+1. Applying software environments
+   - explain the value of reproducibility with containerized software environments
+   - select a pre-configured image container for your hub
+   - pull a custom container to your hub
+1. Navigating the filesystem and transferring data to and from the hub
+   - locate where to store your datasets (home versus shared directory)
+   - transfer files to and from your hub
+1. :star: Troubleshooting and providing user support
+   - triage common problems and know where to look for help
+   - share learning experiences from self-guided study
+   - foster a sustainable support system for your community
+
+(:star: indicates this episode is delivered via a synchronous workshop)
+
+## Prerequisites
+
+Before joining the training, participants should be able to:
+
+- access their hub
```

Comment:
change to 'access the 2i2c training hub' ? this would standardized the experience for all learners in the same cohort. It also means that you can take the training even if you are not already set up as a hub champion.

Reply:
I agree, providing a standard training environment is the way to go!